### PR TITLE
Chris/eval

### PR DESCRIPTION
**Pull request by ChrisWoo0443**
{"exact": 62.26732923439737, 
"f1": 62.84439877221093, 
"total": 11873, 
"HasAns_exact": 37.48313090418353,
 "HasAns_f1": 38.63892486883597, 
"HasAns_total": 5928, 
"NoAns_exact": 86.98065601345668,
 "NoAns_f1": 86.98065601345668,
 "NoAns_total": 5945,
 "best_exact": 62.26732923439737,
 "best_exact_thresh": 0.4999566674232483,
 "best_f1": 62.844398772211, 
"best_f1_thresh": 0.4999566674232483,
 "pr_exact_ap": 28.850294282763304, 
"pr_f1_ap": 30.013548497054497,
 "pr_oracle_ap": 72.5175008162682}

So far our model does a little worse than other distilbert models finetuned on squadv2. I will update the dataset Preprocessing to tokenize only the train and validation split and remove the test split. 
Another factor could also be the learning rate or how we actually go about finetuning our model. 

I found this repo: https://github.com/fastforwardlabs/ff14_blog/blob/master/_notebooks/2020-05-19-Getting_Started_with_QA.ipynb that finetunes using huggingfaces provided script and their results were better than ours
([('exact', 66.25958056093658),
             ('f1', 69.66994428499025),
             ('total', 11873),
             ('HasAns_exact', 68.91025641025641),
             ('HasAns_f1', 75.74076391627662),
             ('HasAns_total', 5928),
             ('NoAns_exact', 63.61648444070648),
             ('NoAns_f1', 63.61648444070648),
             ('NoAns_total', 5945),
             ('best_exact', 68.36519834919565),
             ('best_exact_thresh', -4.189256191253662),
             ('best_f1', 71.1144383018176),
             ('best_f1_thresh', -3.767639636993408)])

### CODE REVIEW

**Review by HivaMohammadzadeh1 — APPROVED** [review]
Looks good to me!
